src/System.Private.Interpreter/src/Internal/Runtime/Interpreter/ILImporter.Interpreter.cs

```diff
@@ -239,22 +255,1021 @@ private void ImportLoadFloat(double value)
+                            }
+                            else if (opcode == ILOpcode.cgt_un)
+                            {
+                                result = val2 > val1;
+                            }
+                            else if (opcode == ILOpcode.clt)
+                            {
+                                result = (double.IsNaN(val1) || double.IsNaN(val2)) ? false : val2 < val1;
+                            }
+                            else if (opcode == ILOpcode.clt_un)
+                            {
+                                result = val2 < val1;
+                            }
+                        }
+                        else
+                        {
+                            ThrowHelper.ThrowInvalidProgramException();
+                        }
+                    }
+                    break;
+                case StackValueKind.ObjRef:
+                    {
+                        object val1 = op1.AsObjectRef();
+
+                        if (op2.Kind == StackValueKind.ObjRef)
+                        {
+                            object val2 = op2.AsObjectRef();
+
+                            if (opcode == ILOpcode.ceq)
+                            {
+                                result = val1 == val2;
+                            }
+                            else
+                            {
+                                // TODO: Find GC addresses of objects and compare them
+                            }
+                        }
+                        else
+                        {
+                            ThrowHelper.ThrowInvalidProgramException();
+                        }
+                    }
+                    break;
+                case StackValueKind.ByRef:
+                    // TODO: Handle ByRef scenarios
+                    break;
+                case StackValueKind.ValueType:
+                case StackValueKind.Unknown:
+                default:
+                    ThrowHelper.ThrowInvalidProgramException();
+                    break;
+            }
+
+            _interpreter.EvaluationStack.Push(StackItem.FromInt32(result ? 1 : 0));
         }
 
         private void ImportConvert(WellKnownType wellKnownType, bool checkOverflow, bool unsigned)
         {
-            throw new NotImplementedException();
+            StackItem stackItem = PopWithValidation();
+            switch (wellKnownType)
+            {
+                case WellKnownType.SByte:
+                    {
+                        sbyte result = default(sbyte);
+                        if (stackItem.Kind == StackValueKind.Int32)
+                        {
+                            if (unsigned)
+                            {
+                                uint value = (uint)stackItem.AsInt32();
+                                result = checkOverflow ? Convert.ToSByte(value) : (sbyte)value;
```

Comment:
`checked((sbyte)value)` gives exactly the semantic we need here (including the exception message). `Convert.ToSByte` is a bit different (comment also applicable elsewhere).